apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -262,7 +266,18 @@ public void close() throws Exception {
         channel.shutdown();
       }
       if (!channel.isTerminated()) {
-        channel.awaitTermination(5, TimeUnit.SECONDS);
+        try {
+          channel.awaitTermination(1, TimeUnit.MINUTES);
```

Comment:
is this wait period just for the sinks to finish writing to the database?

Reply:
No this is waiting for the gRPC stub to finish closing its connection to the RDA API server and shut down any worker threads it was using.

Reply:
I bumped the timeout while making this change because I felt 5 seconds was just too little.  The choice of one minute was arbitrary but is well within the time that the pipeline scheduler allows jobs to consume.  The actual time taken will generally be much shorter since the call will return as soon as the shutdown is completed.